web/src/engine/websites/GalaxyManga.ts

```diff
@@ -0,0 +1,90 @@
+import { Tags } from '../Tags';
+import icon from './GalaxyManga.webp';
+import { Chapter, DecoratableMangaScraper, Manga, Page, type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchJSON, FetchRequest } from '../FetchProvider';
+
+type APIManga = {
+    status: boolean,
+    message: string,
+    serie: {
+        id: number,
+        title:string
+    }
+}
+
+type APIMangas = {
+    data: {
+        id: number,
+        title: string
+    }[]
+}
+
+type APIChapter = {
+    id: number,
+    title: string,
+    name: string
+}
+
+type APIPages = {
+    chapter: {
+        chapterData: {
+            webtoon: string[]
+        }
+    }
+}
+
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+
+    private readonly apiurl = 'https://api.flixscans.com';
+
+    public constructor() {
+        super('galaxymanga', 'Galaxy Manga', 'https://flixscans.com', Tags.Media.Manga, Tags.Media.Manhwa, Tags.Media.Manhua, Tags.Language.Arabic, Tags.Source.Scanlator);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    public override ValidateMangaURL(url: string): boolean {
+        return new RegExp(`^${this.URI.origin}/series/[^/]+$`).test(url);
+    }
+
+    public override async FetchManga(provider: MangaPlugin, url: string): Promise<Manga> {
+        const id = new URL(url).pathname.split('-')[1];
+        const request = new FetchRequest(new URL(`/api/v1/webtoon/series/${id}`, this.apiurl).href);
+        const { serie } = await FetchJSON<APIManga>(request);
+        return new Manga(this, provider, serie.id.toString(), serie.title);
+    }
+
+    public override async FetchMangas(provider: MangaPlugin): Promise<Manga[]> {
+        const mangaList = [];
+        for (let page = 1, run = true; run; page++) {
+            const mangas = await this.getMangasFromPage(page, provider);
+            mangas.length > 0 ? mangaList.push(...mangas) : run = false;
+        }
+        return mangaList;
+    }
+
+    async getMangasFromPage(page: number, provider: MangaPlugin): Promise<Manga[]> {
+        const request = new FetchRequest(new URL(`/api/v1/webtoon/homepage/latest/home?page=${page}`, this.apiurl).href);
+        const { data } = await FetchJSON<APIMangas>(request);
+        return data.map(manga => new Manga(this, provider, String(manga.id), manga.title));
+    }
+
+    public override async FetchChapters(manga: Manga): Promise<Chapter[]> {
+        const uri = new URL(`/api/v1/webtoon/chapters/${manga.Identifier}-desc`, this.apiurl);
+        const request = new FetchRequest(uri.href);
+        const data = await FetchJSON<APIChapter[]>(request);
+        return data.map(chapter => new Chapter(this, manga, chapter.id.toString(), `${chapter.name} ${chapter.title || ''}`.trim()));
```

Comment:
Does it make sense to use chapter name + title?

Reply:
`name `is just the chapter number. `title `is the actual title of the chapter, when its not null. 
If possible, its better to have "40 the rabbit' returns than just '40' for chapter 40.